src/d_main.cpp

```diff
@@ -685,7 +685,7 @@ void D_Display ()
 	if (viewactive)
 	{
 		R_SetFOV (r_viewpoint, players[consoleplayer].camera && players[consoleplayer].camera->player ?
-			players[consoleplayer].camera->player->FOV : 90.f);
+			players[consoleplayer].camera->player->FOV : players[consoleplayer].camera->CameraFOV);
```

Comment:
The value of `players[consoleplayer].camera` can `nullptr` in this line so fallback to 90 degrees is still required.
I think `R_SetFOV()` call needs to rewritten using intermediate variables and proper checks.

Reply:
Done.